### DISCUSSION TITLE
Can I add a label at the top level.

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#878] [Q&A] [answered]
---
title: How Supervised Learning Works ?
description: In this section you will learn how supervised learning works.
label: <kbd> Beginner <kbd>
---

in this file is there any way that I can add optional syntax of my own along with this two components ? and icon.

**Comment by Gitstar-OC** [comment]
```
---
title: How Supervised Learning Works ?
description: In this section you will learn how supervised learning works.
label: <kbd> Beginner <kbd>
---
```

**Comment by fuma-nama** [comment, marked as answer]
You cannot have JSX in frontmatter. Put the rendering logic to page.tsx instead

**Comment by Gitstar-OC** [comment]
Is there a way to add comments in the mdx file ?

**Reply by fuma-nama** [reply]
```mdx
{/* like this? */}
```
